src/functions.php

```diff
@@ -12,22 +12,29 @@
  * Creates a `Promise` which resolves with the stream data buffer
  *
  * @param ReadableStreamInterface $stream
+ * @param int $maxLength
```

Comment:
`@param int|null $maxLength maximum number of bytes to buffer or null=unlimited`? See also signature in README.